fe2-android/app/src/main/java/com/github/dedis/popstellar/model/objects/security/PublicKey.kt

```diff
@@ -14,13 +15,16 @@ import timber.log.Timber
 @Immutable
 class PublicKey : Base64URLData {
   private val verifier: PublicKeyVerify
+  private var toUsername: String
```

Comment:
Just a personal preference: maybe in the context of PublicKey I'd choose another name for that instead of username, such as keyLabel or simply label

Reply:
Also, do you want this field to be serialized? Because if not you should add a @Transient annotation

Reply:
In fact the field should not be serialized. As it is deterministic, no need to send in anywhere since any system should be able to re-compute it in local themselves.